src/features/collaborator-avatars/constants.js

```diff
@@ -0,0 +1,7 @@
+// @flow
+
+const COLLAB_GROUP_TYPE = 'group';
```

Comment:
Nitpick, normally I would go from common to specific. So GROUP_TYPE_XXX. Sometimes easier to search (and replace if need be) all constants relating to collab types by searching for COLLAB_TYPE_

But this is fine too.